ehitchhiking-ui/src/app/shared/components/pre-loading/pre-loading.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+@Component({
+	selector: 'app-pre-loading',
+	templateUrl: './pre-loading.component.html',
+	styleUrls: ['./pre-loading.component.sass'],
+})
+export class PreLoadingComponent implements OnInit {
+	@Input() isLoading: boolean;
+	@Input() size: string;
```

Comment:
use a created enum here + set default value. Kindly look at TypeScript docs how to do that

Reply:
done